### PR TITLE
detection report viewer

### PR DESCRIPTION
**Pull request by yoouung**
- frontend
  - modify stylesheet link in html template
  - add pdf frame in detection report page
- docs
  - update README.md for additional library install

### CODE REVIEW

**Comment by yoouung** [comment]
image

File: dt-demo-fe/src/pages/filtering/detectionReport.js [hunk 1]
```diff
@@ -58,18 +59,78 @@ const DetectionContainer = () => {
             await useApi.post("/filter/detection/type", {
                 type: radioSelected,
             });
-            console.log("detection type", radioSelected, " sent successfully");
+            alert("필터링이 완료되었습니다.");
```
Comment (doorooful) [inline dt-demo-fe/src/pages/filtering/detectionReport.js:62]: Can you confirm the message?

**Review by doorooful — CHANGES_REQUESTED** [review]
Can you confirm the alert message?

File: dt-demo-fe/src/pages/filtering/detectionReport.js (hunk 1 above)
Comment (yoouung) [reply dt-demo-fe/src/pages/filtering/detectionReport.js]: I changed the message!
please check the updated code.
thanks for your check :)

**Review by doorooful — APPROVED** [review]
LGTM!👍

**Pull request merged**